src/module.ts

```
@@ -10,17 +10,20 @@ import { selectNodeTransform } from "./runtime/lib/core/transforms/select-transf
 
 // Module options TypeScript interface definition
 // eslint-disable-next-line @typescript-eslint/no-empty-object-type
-export interface ModuleOptions {}
+export interface CXModuleOptions {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)